src/elements/unified-share-modal-element/__tests__/UnifiedShareModalElement.test.js

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+import API from '../../../api';
+import ErrorMask from '../../../components/error-mask/ErrorMask';
+import UnifiedShareModalElement from '../UnifiedShareModalElement';
+import UnifiedShareModal from '../../../features/unified-share-modal/UnifiedShareModal';
+import { DEFAULT_HOSTNAME_API, TYPE_FILE, TYPE_FOLDER } from '../../../constants';
+import { normalizeItemResponse, normalizeUserResponse } from '../utils';
+
+jest.mock('../../../api');
+jest.mock('../utils');
+
+const SHARED_LINK = {
```

Comment:
Each of these represent the internal data format for the USM component, i think. may be helpful to label them here, so they arent confused with the expected element types. Might also help to pull them into a separate file if they solely exist as testing helper objects.